.github/workflows/test.yml

```
@@ -136,16 +190,17 @@ jobs:
+      - name: Run linting
         run: |
-          flake8 . || echo "⚠️ Flake8 issues found - please fix manually"
-          mypy backend/core --ignore-missing-imports || echo "⚠️ Type checking issues found - please fix manually"
+          flake8 .
+          mypy backend/core --ignore-missing-imports
 
       - name: Security scan
         run: |
-          safety check || echo "⚠️ Security vulnerabilities found - please review"
+          safety check
+        continue-on-error: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)